dtreeviz/trees.py

```diff
@@ -1591,6 +1623,18 @@ def _ctreeviz_bivar(shadow_tree, fontsize, ticks_fontsize, fontname, show,
     _format_axes(ax, shadow_tree.feature_names[featidx[0]], shadow_tree.feature_names[featidx[1]],
                  colors, fontsize, fontname, ticks_fontsize=ticks_fontsize, grid=False)
 
+    if 'splits' in show:
+        plt.draw()
+        print("labels", ax.get_xticklabels(), ax.get_yticklabels())
```

Comment:
@tlapusan is this print for debugging? Probably should be commented out.

Reply:
sorry. yeah, debugging.